Bottom: 75326e339d785a2b0d213aa52a0e9e6115d9cdd8
Top:    14007fa533107d9bd04332416816965b205c1944
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 23:13:10 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
